portfolio.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link href="portstyle.css" type="text/css" rel="stylesheet">
+  
+</head>
+<body>
+
+	<section class="navigationBar">
+		<div class="navigationButtons">
+			<div class="home">
+			<a href="#">Home</a>
+		</div>
+			<div class="about">
+			<a href="#">About</a>
+		</div>
+			<div class="portfolio">
+			<a href="#">Portfolio</a>
+		</div>
+			<div class="socials">
+			<a href="#">Socials</a>
+		</div>
+		</div>
+		
+	</section>
+	
+	<section class="mainImage">
+		<div class="backgroundImage">
+		<div class="textoverlayed">
```

Comment:
I can't work out what this div is trying to do? What are you using it for?

Reply:
Was used previously just hadn't gotten around to deleting it so thank you for pointing it out